lambda/utils.js

```diff
@@ -554,8 +554,13 @@ module.exports.extractDurationFromText = (log) => {
  */
 module.exports.extractDurationFromJSON = (log) => {
     // extract each line and parse it to JSON object
-    const lines = log.split('\n').map((line) => JSON.parse(line));
-
+    const lines = log.split('\n').filter((line) => line.startsWith('{')).map((line) => {
+         try {
+              return JSON.parse(line);
+         } catch (e) {
+             return '';
```

Comment:
So this only happens if a non-JSON line starts with `"{"`, which should be very unlikely.

I would still `console.error(...)` something here, otherwise the silent failure will be very hard to detect later on :)

Something like

```nodejs
console.error(`Detected invalid JSON line: ${line}`);
return '':